src/flows/pipes/Notification/HTTP.tsx

```diff
@@ -5,14 +5,19 @@ import {
   Input,
   InputType,
   ComponentSize,
+  ComponentStatus,
   SelectGroup,
   FlexDirection,
   AlignItems,
 } from '@influxdata/clockface'
 
 import {PipeContext} from 'src/flows/context/pipe'
 
-const HTTP: FC = () => {
+interface Props {
```

Comment:
NO. this should be using the readOnlyComponent mechanism

Reply:
oh yeah I meant to ask about this. I didn't feel super great about doing it this way - but these components are created in a weird fashion in the Notification view and I wasn't sure what we should do about it.